File: INICIANTE/1020.py
idadeDias = int(input())
ano = idadeDias//365
mes = (idadeDias%365)//30
dia = idadeDias%365%30
print('{} ano(s)'.format(ano))
print('{} mes(es)'.format(mes))
print('{:.0f} dia(s)'.format(dia))

File: INICIANTE/1020.c
#include <stdio.h>
#define ANO 365
#define MES 30

void mostra_idade_fatorada(int t_dias){
    printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", t_dias/ANO, (t_dias%ANO)/MES, t_dias%ANO%MES);
}

int main(){
    int total_dias = 0;
    scanf("%d", &total_dias);
    mostra_idade_fatorada(total_dias);
    return 0;
}
